Source/Code/KCD_1042192.DisclaimAcceptanceLog/KCD_1042192.DisclaimAcceptanceLog.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15

```

Comment:
Not many committing and no real guidelines around version for this product, so I think moving to 2017 is OK.